test_data.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
unused?

Personally, I favor more end-to-endy tests (allowing us to detect errors in calling the importer from the management command), but in general, I dislike having test data separately, it always gets messy to maintain those. I'd probably define this as a python dict in the corresponding test file, then.